scripts/hooks/tech-lead-authoring-guard.py

```
@@ -0,0 +1,479 @@
+        return "software-engineer"
+    if norm == "CHANGELOG.md" or norm.startswith("docs/"):
+        return "tech-writer"
+    return "the appropriate specialist"
+
+
+# ---------------------------------------------------------------------------
+# Escape-hatch parsing.
+# ---------------------------------------------------------------------------
+
+
+def _agent_push_role():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_agent_push_role`

(ANN202)